nemo/collections/asr/models/configs/ctc_models_config.py

```diff
@@ -0,0 +1,164 @@
+# Copyright (c) 2020, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from dataclasses import dataclass, field
+from typing import Any, Dict, List, Optional
+
+from omegaconf import MISSING, OmegaConf
+
+from nemo.core.config import modelPT as model_cfg
+
+
+@dataclass
+class BaseASRPreprocessorConfig:
+    pass
```

Comment:
Isn't this redundant?

Reply:
Yes, all the Base* classes are redundant. We can have a placeholder type with Any instead of them, but in hindsight creating empty base classes just is not a good experience so I'll be dropping them in my next iteration.